src/main/java/org/prebid/server/bidder/admatic/AdmaticBidder.java

```diff
@@ -76,6 +81,18 @@ private String resolveEndpoint(AdmaticImpExt impExt) {
         return endpointUrl.replace(HOST_MACRO, HttpUtil.encodeUrl(impExt.getHost()));
     }
 
+    private MultiMap headers(Device device) {
+        final MultiMap headers = HttpUtil.headers();
+
+        if (device != null) {
+            HttpUtil.addHeaderIfValueIsNotEmpty(headers, HttpUtil.X_FORWARDED_FOR_HEADER, device.getIp());
+            HttpUtil.addHeaderIfValueIsNotEmpty(headers, HttpUtil.X_FORWARDED_FOR_HEADER, device.getIpv6());
```

Comment:
Wrong sequence. IpV6 must be first so it will be overwritten by ip if both are present